app/scripts/modules/core/src/pipeline/config/stages/evaluateVariables/EvaluateVariablesExecutionDetails.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { StageExecutionLogs, StageFailureMessage } from 'core/pipeline/details';
```

Comment:
Let’s export all this stuff from index files then import from `core/pipeline`

Reply:
Looks like it's already exported. I'll use `core/pipeline` and fix the other stages in a separate PR